apps.toml

```diff
@@ -2013,6 +2013,11 @@ state = "notworking"
 subtags = [ "pastebin" ]
 url = "https://github.com/YunoHost-Apps/modernpaste_ynh"
 
+[moncycle.app]
```

Comment:
`moncycle.app` is that the name of the app upstream?

Reply:
Yes, it's both the name of the app and the url of the (free to use) official instance. Shall I change something ?

Reply:
The app name should be the ID set in the manifest : `"id": "moncycle_app",`
I propose, to avoide confusion, to drop the `.app` for the `name` and the `ID`. (this is not a requirement of course)

Reply:
Understood, you are right that this `.app` may create some confusion.
What do you think about going with `moncycle_app` as a `name` and `ID` ?
Maybe I should rename the repo accordingly, to `moncycle_app_ynh` : I'm not sure whether I can do it easily or not, I'll check !

Reply:
(Imho i would just name it `moncycle`, having an `_app` suffix is a bit meh, considering that any yunohost app is ... an app)

Reply:
I get your logic, no problem with that 👍 
Let me fix my stupid CI mistake, change the repo and repo fork, amend this PR accordingly and all should be good. Thanks !